opal/runtime/opal_progress.c

```diff
@@ -138,6 +143,8 @@ opal_progress_init(void)
                  opal_progress_event_flag));
     OPAL_OUTPUT((debug_output, "progress: initialized yield_when_idle to: %s",
                  opal_progress_yield_when_idle ? "true" : "false"));
+    OPAL_OUTPUT((debug_output, "progress: initialized poll_when_idle to: %s",
```

Comment:
What does it mean to set both `poll_when_idle` *and* `yield_when_idle` to true?

I'm wondering if we should have a higher-level abstraction name -- i.e., use `yield_when_idle` to mean that the user intends to yield the processor when MPI is idle (regardless of whether we use `yield()` or some other mechanism).  Otherwise, we have 2 mechanisms that do (kinda) the same thing, and one of them may not be supported on all platforms.  I think it would be better to have one MCA param that intelligently tries to give up the processor when MPI is "idle".